### PR TITLE
Add appveyor configuration

### PR DESCRIPTION
**Pull request by EmanueleMinotto**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

Publishing tests results of the demo application on Windows too could be a warranty for some users.
It's not yet based on all the [available PHP distributions for Windows](http://windows.php.net/download/) but for the current active version is passing.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  - cinst -y OpenSSL.Light
+  - cinst -y php
+  - cd c:\tools\php
+  - copy php.ini-production php.ini /Y
+  - echo date.timezone="UTC" >> php.ini
+  - echo extension_dir=ext >> php.ini
+  - echo extension=php_openssl.dll >> php.ini
+  - echo extension=php_mbstring.dll >> php.ini
+  - echo extension=php_intl.dll >> php.ini
+  - echo extension=php_gd2.dll >> php.ini
```
Comment (javiereguiluz) [inline appveyor.yml]: Just asking: why do we need this extension?

Comment (EmanueleMinotto) [inline appveyor.yml]: That's not required, my mistake.
Now I've removed it.

**Comment by javiereguiluz** [comment]
@EmanueleMinotto thanks for working in this. I'll wait a bit before merging it so people that understand this configuration can review it. Thanks.

**Comment by EmanueleMinotto** [comment]
@javiereguiluz sure, I'm leaving appveyor output here for a fast access to it: https://ci.appveyor.com/project/EmanueleMinotto/symfony-demo

File: appveyor.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  - echo extension_dir=ext >> php.ini
+  - echo extension=php_openssl.dll >> php.ini
+  - echo extension=php_mbstring.dll >> php.ini
+  - echo extension=php_intl.dll >> php.ini
+  - echo extension=php_pdo_sqlite.dll >> php.ini
+  - echo memory_limit=1G >> php.ini
+  - cd %APPVEYOR_BUILD_FOLDER%
+  - php -r "readfile('http://getcomposer.org/installer');" | php
+  - php composer.phar install --no-interaction --no-progress
+  - ps: (new-object net.webclient).DownloadFile('https://phar.phpunit.de/phpunit.phar', ${env:APPVEYOR_BUILD_FOLDER} + '\phpunit.phar')
```
Comment (stof) [inline appveyor.yml]: this can be simplified:

```
appveyor DownloadFile https://phar.phpunit.de/phpunit.phar
```

**Comment by stof** [comment]
@javiereguiluz I suggest you to enable AppVeyor on the repo *before* merging this (configuring it to run only when there is a config file, so that it does not impact other PRs). This way, we would have the feedback in the PR status

**Comment by javiereguiluz** [comment]
@fabpot when you have some free time, please enable AppVeyor for this `symfony-demo` repository. Thanks!

**Comment by fabpot** [comment]
done

**Comment by stof** [comment]
@EmanueleMinotto please rebase your PR so that it triggers an AppVeyor build now that it is enabled on the repo

**Comment by EmanueleMinotto** [comment]
@javiereguiluz PR rebased, now should be ok

**Comment by javiereguiluz** [comment]
@EmanueleMinotto thanks. By the way, there are some warnings in the AppVeyor output:

```
WARNING: Write-ChocolateySuccess is deprecated. If you are the maintainer, 
please remove it from your package file.

...

Get-BinRoot is going to be deprecated by v1. Many packages no longer require
it since the folders no longer have versions on them.

...
```

**Comment by stof** [comment]
@javiereguiluz this should be reported to the maintainers of these chocolatey packages

**Comment by javiereguiluz** [comment]
@stof merging then. Thanks.

**Pull request merged**